colcon_core/shell/template/prefix_util.py.em

```diff
@@ -295,11 +304,7 @@ def _prepend_unique_value(name, value):
             env_state[name] = set()
         if os.environ.get(name):
             env_state[name] = set(os.environ[name].split(os.pathsep))
-    if not env_state[name]:
-        extend = ''
-    else:
-        extend = os.pathsep + FORMAT_STR_USE_ENV_VAR.format_map(
-            {'name': name})
+    extend = os.pathsep + FORMAT_STR_USE_ENV_VAR.format_map({'name': name})
```

Comment:
Maybe add a comment mentioning that `_remove_trailing_separators` is going to clean up the unintentional trailing one.

Reply:
Added de5afe2